app/src/main/java/org/xbmc/kore/Settings.java

```diff
@@ -53,6 +53,10 @@
     public static final String KEY_PREF_SHOW_NOTIFICATION = "pref_show_notification";
     public static final boolean DEFAULT_PREF_SHOW_NOTIFICATION = false;
 
+    // Pause during calls
+    public static final String KEY_PREF_PAUSE_DURING_CALLS = "pref_pause_during_calls";
+    public static final boolean DEFAULT_PREF_PAUSE_DURING_CALLS = true;
```

Comment:
Well I think it shouldn't. Difference of opinion will always be present on options so I'd agree with @SyncedSynapse